CHANGELOG.md

```diff
@@ -335,6 +335,12 @@ Deprecated names
   sym-↔   ↦   ↔-sym
   ```
 
+* In `Data.Fin.Base`: two new, hopefully more memorable, names `↑ˡ` `↑ʳ` for the 'left', resp. 'right' injection of a Fin m into a larger Fin (n + m), with argument order to reflect the position of the Fin m argument. Knock-on changes in `Data.Fin.Properties`, `Data.Vec.Properties`, `Data.Vec.Functional`, `Data.Vec.Functional.Properties`, `Data.Vec.Functional.Relation.Binary.Pointwise.Properties`, `Data.Vec.Functional.Relation.Unary.All.Properties`. 
```

Comment:
No need to list all the modules touched, but it would be great if you could list all the deprecated proof names as well.

Reply:
Re: deprecated proof names: on  a per-module basis? or just for `Data.Fin.Base`?

Reply:
On a per-module basis, so I think we need an entry list for `Data.Vec.Properties` and one more module which has temporarily slipped my mind?

Reply:
Done. It seems as though only three modules were touched, for deprecation. 
All the others are about using the new names/argument orders.